### ISSUE TITLE
Homebrew now installing v1 instead of v2

### ISSUE THREAD

**Issue by bdrelling** [getsentry/sentry-cli#1803]
### Environment

Xcode Cloud
macOS 13.4.1

### Steps to Reproduce

Running `brew install getsentry/tools/sentry-cli` on CI machine used to install 2.x, now it's installing 1.x after the release 4 days ago.

### Expected/Actual Result

We'd keep getting the latest version, not the latest tag.

For what it's worth, we're going to fix this in the meantime by pinning the version as mentioned in the [Installation](https://github.com/getsentry/sentry-cli#installation) steps, but we'd prefer to keep using Homebrew moving forward if possible.

**Comment by loewenheim** [comment]
Hi, sorry for the mishap. Can you verify if this still happens?

**Comment by theophileds** [comment]
Hey there,

I have the same issue.
`❯ brew install getsentry/tools/sentry-cli
Warning: getsentry/tools/sentry-cli 1.76.0 is already installed and up-to-date.
To reinstall 1.76.0, run:
  brew reinstall sentry-cli`

But even more concerning, our CI is broken as well, as we rely on the gh-action [setup-sentry-cli](https://github.com/mathieu-bour/setup-sentry-cli#version) to install the latest version of Sentry-cli.

The CI output is:
`sentry-cli 1.76.0
error: Found argument 'sourcemaps' which wasn't expected, or isn't valid in this context

USAGE:
    sentry-cli <SUBCOMMAND>
`

**Comment by loewenheim** [comment]
We just released version 2.21.3.
```
❯ brew info getsentry/tools/sentry-cli
==> getsentry/tools/sentry-cli: stable 2.21.3
```
